Create consent.mdx (#801)
xmtp/xmtp-dot-org: src/pages/consent.mdx

Today, individual apps implement their own ways of handling consent granting - through app-specific invite links, group joining mechanisms, or eligibility rules. We are creating an interoperable standard that works across the entire ecosystem:

- Apps can generate and validate consent links using a common protocol
- Links work consistently across all XMTP apps
- Supports flexible eligibility criteria (token ownership, DAO membership, etc.)
- Enables ecosystem-wide features like group invites and automated consent rules
- Maintains privacy while improving user experience

### 2. Network Enforcement

Moving from client-side filtering to network-level enforcement:

- Network validates consent before delivering messages
- Reduces network load from unwanted messages
- Provides stronger spam prevention
- Maintains end-to-end privacy

More details about network enforcement will be shared as development progresses.

## Frequently Asked Questions

1. **If I block someone, will they know?**

    No, blocked users aren't notified of their status. From their perspective, it appears the same as if you haven't responded to their message.

2. **Can I revoke consent after granting it?**

>    Yes, you can always privately revoke consent. The other party won't be notified, but they'll no longer be able to message you.

Reviewer: Suggesting a slight tweak here - I think the other party can still message you, but you won't see their messages. 

```suggestion
   Yes, you can always privately revoke consent. The other party won't be notified, but you'll no longer see their messages.
```
